internal/core/worker/evolution/worker_test.go

```diff
@@ -225,7 +225,7 @@ func TestScanEvoChainOnce(t *testing.T) {
 			client, scanner := getMocks(t)
 			state, tx2 := getMocksFromState(t)
 
-            evoWorker := NewWorker(&tt.c, client, scanner, state)
+			evoWorker := NewWorker(&tt.c, client, scanner, state)
```

Comment:
similar as for worker

Reply:
As agreed in private, this change will stay in the PR